moped-editor/src/views/projects/projectsListView/ProjectsListViewQueryConf.js

```diff
@@ -75,18 +72,6 @@ export const ProjectsListViewQueryConf = {
       },
       width: "*",
       type: "String",
-      filter: (values) => {
```

Comment:
this was used in the grid table row rendering, im removing it since I removed the `singleItem` parameter from gql abstract